pom.xml

```diff
@@ -163,6 +163,7 @@
         <module>debezium-connector-sqlserver</module>
         <module>debezium-microbenchmark</module>
         <module>debezium-quarkus-outbox</module>
+        <module>debezium-quarkus-outbox-integration-tests</module>
```

Comment:
Would it make sense to actually move is *into* `debezium-quarkus-outbox`, next to `runtime` and `deployment`?

Reply:
That's a good question, it would definitely make sense from our hierarchy structure.  I'll take a look at it.